v1structs/v1structs.go

```diff
@@ -84,6 +84,19 @@ type SecretMetadata struct {
 	SecretUUID string `json:"secretUuid"`
 }
 
+// UpsertTeamRequest is the JSON structure containing a signed team roster.
+type UpsertTeamRequest struct {
+	// TeamRoster describes the members and configuration of a team.
+	// See github.com/fluidkeys/fluidkeys/teamroster
```

Comment:
This results in a 404. Should you include `/blob/master`?

Reply:
This is only the struct, I haven't made the endpoint yet !!!

Reply:
My mistake, I interpretted this as a web url.